package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@firebolt-js/openrpc",
+  "type": "module",
```

Comment:
## This needs attention!

Jest needed me to change the `type` property in the `package.json` to `module`. The Jest docs [_saaaay_](https://jestjs.io/docs/ecmascript-modules) that they attempt to detect ECMAScript Modules that same way that NodeJS does, but that isn't true. 

> _Beyond that, we attempt to follow node's logic for activating "ESM mode" (such as looking at type in package.json or .mjs files)_

Jest requires both `.mjs` extension **AND** the `package.json` change to activate ESM mode.

[NodeJS](https://nodejs.org/api/esm.html#esm_enabling), on the other hand, will OR the following:

- Presence of the `.mjs` file extension
- `"type": "module"` in `package.json`
- `--input-type="module"` runtime flag

Reply:
I tried this a long time ago and it caused problems I can't recall... things have evolved quite a bit since then, so I think your "This needs attention" is still true :)